hypothesis-python/src/hypothesis/strategies.py

```diff
@@ -2062,4 +2063,19 @@ def deferred(definition):
     return DeferredStrategy(definition)
 
 
+@defines_strategy_with_reusable_values
+def emails():
+    """A strategy for email addresses.
+
+    See https://github.com/HypothesisWorks/hypothesis-python/issues/162
+    for work on a permanent replacement.
+    """
+    from hypothesis.provisional import domains
+    local_chars = string.ascii_letters + string.digits + "!#$%&'*+-/=^_`{|}~"
+    local_part = text(local_chars, min_size=1, max_size=64)
+    # TODO: include dot-atoms, quoted strings, escaped chars, etc in local part
+    return builds('{}@{}'.format, local_part, domains()).filter(
```

Comment:
To make this *obviously* unicode on any version of python, add the `u` prefix (giving `u'{}@{}'.format`).